incubator/schema-registry-ui/README.md

```diff
@@ -0,0 +1,74 @@
+# Schema-Registry-UI Helm Chart
+This helm chart creates a [Schema-Registry-UI server](https://github.com/Landoop/schema-registry-ui).
+
+This is a web tool for the confluentinc/schema-registry in order to create / view / search / evolve / view history & configure Avro schemas of your Kafka cluster.
+
+
+## Prerequisites
+* Kubernetes 1.6
+* A running Kafka Installation
+* A running Zookeeper Installation
+* A running Schema-Registry installation
+
+## Chart Components
+This chart will do the following:
+
+* Create a Schema-Registry-UI deployment
+* Create a Service configured to connect to the available Schema-Registry-UI pods on the configured
+  client port.
+
+## Installing the Chart
+You can install the chart with the release name `srui` as below.
+
+```console
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install --name srui incubator/schema-registry-ui
+```
+
+If you do not specify a name, helm will select a name for you.
+
+### Installed Components
```

Comment:
I'd remove this section. `helm status <release>` is probably the better choice anyways.